v1/runtime/runtime_test.go

```diff
@@ -1695,3 +1701,141 @@ func TestExtraDiscoveryOpts(t *testing.T) {
 		t.Error("expected extra discovery option to be called")
 	}
 }
+
+type factory struct {
+	m *plugins.Manager
+}
+
+func (f *factory) New(m *plugins.Manager, _ any) plugins.Plugin {
+	m.ExtraRoute("GET /v1/flusher", "v1/flusher", func(w http.ResponseWriter, _ *http.Request) {
```

Comment:
ℹ️ We're using ExtraRoute here to ensure that our test method is wired up with the same instrumentation as OPA's other handlers.